### DISCUSSION TITLE
Submit Project Button Not visible.

### DISCUSSION THREAD

**Discussion by tushargupta21** [github/india#827] [Students]
I had previously submitted my project but I got a mail from FUTURE READY TALENT to re-submit it. Now I am not able to re-submit it as "Save and Submit" button/option is not visible. Please help me. @ey-mentor1,  @EY-Mentor3, @ey-mentor2  , @EY-Mentor4

**Comment by ey-mentor1** [comment]
Hello,
Can you please attach the screenshot, so that we can look into the issue.
Thank you

**Reply by tushargupta21** [reply]
Yes, These are the screenshots
![Screenshot (1075)](https://user-images.githubusercontent.com/95370344/161275162-12d72e5f-ee4f-48a9-aea8-6274b856730d.png)
![Screenshot (1076)](https://user-images.githubusercontent.com/95370344/161275172-81376496-583b-41be-8c79-3aff274b2c90.png)

**Reply by Baswala** [reply]
I have completed my self learning, industry session, and also update Microsoft with Microsoft Azure skills  but I am not get submit button sir

**Reply by 555subhasri** [reply]
Same as my problem... please say solution

**Reply by RishiPratap** [reply]
> Hello, Can you please attach the screenshot, so that we can look into the issue. Thank you

@ey-mentor1 similar my project submission button is also not visible please help regarding this.

<img width="849" alt="image" src="https://user-images.githubusercontent.com/72687585/195168293-e0ad09c8-9748-4629-b476-7384d9d60267.png">

I have also mailed you regarding this..

**Reply by yukesh1717** [reply]
I too have same problem.... Can anyone please help me to fix the problem.... My project deadline is near

**Reply by GOUSEPEERA6267** [reply]
> I have completed my self learning, industry session, and also update Microsoft with Microsoft Azure skills but I am not get submit button sir

Hi ...Did you get the submission button..I was trying a lot ...it was showing my Azure skills are under validation...any solution

**Reply by yukesh1717** [reply]
no sir,it still does'nt activated

**Reply by yukesh1717** [reply]
can anyone please reply

**Comment by tushargupta21** [comment]
Yes, These are the screenshots @ey-mentor1 .
Please Help me
![Screenshot (1075)](https://user-images.githubusercontent.com/95370344/161369877-16ee9f90-ee68-46e7-b76e-c6bece4b6fc9.png)
![Screenshot (1076)](https://user-images.githubusercontent.com/95370344/161369883-fcdb75e2-1ed9-4959-8b68-94f3b92e604e.png)

**Reply by ey-mentor1** [reply]
Hello,
Let me check and get back to you.
It may take this weeks time, please bear.
Thank you

**Reply by tushargupta21** [reply]
Ok Sir. Thank you @ey-mentor1

**Reply by BellamShanthiSree** [reply]
Sir I didn't submit my future ready talent as it got the project deadline is exceeded due to not claimed the student developer pack. Could you please help me for getting Microsoft internship certificate sir.

**Comment by ey-mentor1** [comment]
Hello,
The last date for the submission is expired. Due to that reason you are be getting this issue.
Backend team will be sharing the emails to the eligible students as per batch there batch. Once you receive the mail, post that only you can save and submit the project.
Thank you

**Comment by Pradipmore12** [comment]
hello I am trying to submit my project today but its showing submission deadline exceeds.
but few days ago they've shown submission date as 15 jun 22.
how can i submit my project now. Please help me. @ey-mentor1,@ey-mentor2, @ey-mentor1 @Ey-mentor3

**Reply by ey-mentor1** [reply]
Hello,
Since it's issue regarding expiration of submission date.
Please drop mail to FRT support team with all the details and account details.
They will look into it.
Thank you

**Reply by Pradipmore12** [reply]
I have already mailed them but no response
Please look into this .@ey-mentor1

**Reply by ey-mentor1** [reply]
Please send it again.
And share the mail details gere

**Reply by ey-mentor1** [reply]
*here

**Reply by Pradipmore12** [reply]
![Screenshot_20220616_111234](https://user-images.githubusercontent.com/90758301/173999844-8cd0a11d-811b-4bf3-90fd-108ab49d285e.jpg)

**Reply by Pradipmore12** [reply]
Thanks for your support @ey-mentor1 .
My issue is resolved now.

**Reply by amanpreet-5045** [reply]
Can you plzz help me out with the same.

**Comment by Pradipmore12** [comment]
![Screenshot (52)](https://user-images.githubusercontent.com/90758301/173896465-b6fb5248-baae-45ec-a7c9-268d9d105353.png)

**Comment by jalagrawal** [comment]
Team I am unable to submit my project, the cause it is showing is that I have not uploaded Microsoft azure as skill on LinkedIn. I Has already updated Microsoft Azure skills on linkedin 10 days ago
this is my LinkedIn profile:     - https://www.linkedin.com/in/jal-agrawal-9754a81a6

My Ms Learn profile:      https://docs.microsoft.com/en-us/users/JalAgrawal-3676

Email : [jalagrawal1713@gmail.com](mailto:jalagrawal1713@gmail.com)
please help me , I have already uploaded Microsoft azure as a skill but future ready is not showing me a submit button.
Plz help me and solve my problem.
Plz help me and solve my problem.

I had also uploaded screenshot
And also this updated LinkedIn screenshot with azure skills
![Screenshot (91)](https://user-images.githubusercontent.com/98449543/178761824-add53f6a-45cb-48dc-a674-48c4154df866.png)
![Screenshot (90)](https://user-images.githubusercontent.com/98449543/178761835-1ad0ff5c-57bc-4bc6-8d6a-f86fc89a4405.png)

**Reply by mahajangaurav22** [reply]
I am getting the same problem...did you got any way to resolve this..

**Comment by shubhp95** [comment]
I am not able to resubmit my project. I got the mail from frt today.
![Screenshot (3)](https://user-images.githubusercontent.com/104738928/180805880-0137917d-ed62-4207-a2f7-2c88d483fde6.png)
![Screenshot (2)](https://user-images.githubusercontent.com/104738928/180805985-3aa23903-38c4-4e8b-85d0-612ff7415b0c.png)

**Comment by shubhp95** [comment]
Its due to the date of submission. The project submission date is expired.

**Comment by mahajangaurav22** [comment]
The date of my project submission is 31st October. I don’t think it is the problem.

**Comment by shubhp95** [comment]
have you got the mail for the resubmission?

**Comment by Baswala** [comment]
Mail open avvatam ledhu naku nv sent chesindhi

**Comment by 555subhasri** [comment]
Miku solution telsa em cheyalo

**Comment by shubhp95** [comment]
Check your project submission date on the home page. If it's expired than
that's the reason of the submission button.

**Comment by geethaakshaya2k3** [comment]
Submit button is not activating

**Reply by yukesh1717** [reply]
Same problem

**Comment by BellamShanthiSree** [comment]
Have you claimed the student developer pack. Then submission button will
get activated.

**Reply by yukesh1717** [reply]
yes i have claimed

**Reply by yukesh1717** [reply]
but it didnt opened

**Comment by syedsami1** [comment]
Hello sir /madam
I completed the frt project synopsis and updated the microsoft azure skills.but project submission is not activated yet .. deadline is in 3 days . I request to approve and update the status of my FRT page.
Thanking you
Please help me. @ey-mentor1,@ey-mentor2, @ey-mentor1 @EY-Mentor3

**Comment by Abhisheknik** [comment]
Hello sir /madam
I completed the frt project synopsis and updated the microsoft azure skills.but project submission is not activated yet .. deadline is in 3 days . I request to approve and update the status of my FRT page.
Thanking you
Please help me. @ey-mentor1,@ey-mentor2, @ey-mentor1 @EY-Mentor3

**Comment by yukesh1717** [comment]
What is the procedure to update Microsoft azure skills

**Comment by prakashsingit** [comment]
Submit Button is not showing. 
My project synopsis is accepted, but project submit button is not visible as it is showing project deadline is exceeded.
I have also mailed regarding this issue.
Kindly look into this matter..  @ey-mentor1 , @ey-mentor2 , @ey-mentor1 ,  @EY-Mentor3 .

![mail](https://user-images.githubusercontent.com/67994762/223462143-fe6d768f-b3b5-4e74-8dcf-12d4b284690d.png)
![1](https://user-images.githubusercontent.com/67994762/223462266-e52fb511-d998-4012-bc15-3ec7d39503aa.png)
![2](https://user-images.githubusercontent.com/67994762/223462281-ee6283ff-165f-4ce5-824a-777445b2216e.png)

**Reply by Sroyals** [reply]
did u got any help I am facing the same issue but now it is showing project deadline exceeds

**Comment by amanpreet-5045** [comment]
I am too the same issue , the submit button for my project submission is not visible.
Please help me. @ey-mentor1 ,@ey-mentor2 , @ey-mentor1 ,@EY-Mentor3

![Screenshot 2023-04-13 121304](https://github.com/github/india/assets/84512609/ed33c482-d6ec-4ea4-b3d6-82625bccfd8c)
![Screenshot 2023-04-13 121353](https://github.com/github/india/assets/84512609/52ddd608-45d4-4a9c-9a50-53d30e96c6cb)

**Reply by Sroyals** [reply]
did u got any help I am facing the same issue but now it is showing project deadline exceeds

**Reply by amanpreet-5045** [reply]
Ya surely they helped me.You just to send a request mail to them , they will definitely find u away out for completing your internship.

**Comment by Sroyals** [comment]
I sent them mail yesterday and today it is showing project deadline
exceeded is there any possibility that they will help me because the
deadline expired.

**Comment by chakrabortyshovik** [comment]
I am  facing the issue, Project synopsis deadline is exceeded. Please help me  to submit the project  
Please help me. @ey-mentor1,@ey-mentor2, @ey-mentor1 @EY-Mentor3
![image](https://github.com/github/india/assets/113231338/6c805012-f18d-4157-961b-e76bb7c921af)
